tools/oav-traffic-converter/ci.yml

```diff
@@ -0,0 +1,122 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - tools/oav-traffic-converter
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - tools/oav-traffic-converter
+
+variables:
+  NodeVersion: '16.x'
+  VAR_ARIFACT_NAME: "drop"
+  VAR_BUILD_ARTIFACT_STAGING_DIRECTORY: $(Build.ArtifactStagingDirectory)
+  NugetSecurityAnalysisWarningLevel: 'none'
+
+stages:
+  - stage: InstallAndTest
+    jobs:
+      - job: Test
+        strategy:
+          matrix:
+            linux:
+              imageName: 'ubuntu-latest'
+            mac:
+              imageName: 'macos-latest'
+            windows:
+              imageName: 'windows-latest'
+        pool:
+          vmImage: $(imageName)
+        steps:
+        - task: NodeTool@0
+          inputs:
+            versionSpec: '$(NodeVersion)'
+          displayName: 'Install Node.js'
+
+        - script: |
+            npm install
+          displayName: 'npm install'
+          workingDirectory: $(System.DefaultWorkingDirectory)/tools/oav-traffic-converter
+
+        - script: |
```

Comment:
I think it will be better to group below steps under if condition here to make this more clearer. Currently you are publishing artifact only if image is ubuntu using step condition. But let's chat about this.

Reply:
@sima-zhu this is another example of an NPM tool that we need to have introduce a common template for publishing npm packages for our tools repo.

Reply:
Will merge these two.

Reply:
For reference this is the issue @sima-zhu filed https://github.com/Azure/azure-sdk-tools/issues/2153 to have common publishing.